### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by pemmasanikrishna**
add elastic search max file descriptors [4096] for elasticsearch process is too low, increase to at least [65535] error resolution.

### CODE REVIEW

**Comment by feng-tao** [comment]
could you help to fix the CLA?

**Comment by pemmasanikrishna** [comment]
@feng-tao  done

File: docs/installation.md [hunk 1]
```diff
@@ -55,11 +55,18 @@ Atlas would be ready once you'll have the following output in the docker output
...
     Traceback (most recent call last):
       File "/home/ubuntu/amundsen/amundsendatabuilder/venv/lib/python3.6/site-packages/neobolt/direct.py", line 831, in _connect
         s.connect(resolved_address)
     ConnectionRefusedError: [Errno 111] Connection refused
 ```
+4. If `elastic search` container stops with an error `max file descriptors [4096] for elasticsearch process is too low, increase to at least [65535]`, then add the below code to the file `docker-amundsen-local.yml` in the `elasticsearch` definition.
+```
+ ulimits:
+   nofile:
+     soft: 65535
```
Comment (feng-tao) [inline docs/installation.md:69]: do you want to increase it in `docker-amundsen-local.yml` file as well? We could do it in a separate pr. Thanks.

Comment (pemmasanikrishna) [reply docs/installation.md:69]: Sure, raising PR
